notebooks/03-query/05-lookup-essentials.ipynb

```diff
@@ -0,0 +1,1560 @@
+    "\"TYR\": \"Tyler Pounds Regional Airport\",\n",
+    "\"TYS\": \"McGhee Tyson Airport\",\n",
+    "\"VLD\": \"Valdosta Regional Airport\",\n",
+    "\"VPS\": \"Destin-Ft Walton Beach Airport\",\n",
+    "\"WRG\": \"Wrangell Airport\",\n",
+    "\"XNA\": \"Northwest Arkansas Regional Airport\",\n",
+    "\"YAK\": \"Yakutat Airport\",\n",
+    "\"YUM\": \"Yuma MCAS/Yuma International Airport\"\n",
+    "                    }\n",
+    "            }\n",
+    "        }\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "postLookup(lookup2_definition)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e115e479-edc9-4353-8a09-6b6c26a5e668",
+   "metadata": {},
+   "source": [
+    "As before, run the next cell to wait for the table to be broadcast around the cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2a011705-4c06-4a3b-ad74-659e75647d7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "waitForLookup(lookup_tier, lookup2_name, 120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "790f5278-c8a7-4e14-ac8c-a16975ccc291",
+   "metadata": {},
+   "source": [
+    "Run the following cell to see the results of a SQL query using both lookups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4588407-161f-4b29-b36d-8d78f4c00796",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    b.v AS \"airportSize\",\n",
+    "    c.v AS \"airportName\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(a.Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\" a\n",
+    "JOIN lookup.\"'''+lookup_name+'''\" b ON a.Origin = b.k\n",
+    "JOIN lookup.\"'''+lookup2_name+'''\" c ON a.Origin = c.k\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "AND b.v = 'small_airport'\n",
+    "GROUP BY 1,2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5cfb7876-edd6-4132-9f13-a680cc6e6c08",
+   "metadata": {},
+   "source": [
+    "Let's run a query on the `INFORMATION_SCHEMA` to see the two tables in the `lookup` schema. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea96371b-d628-40c5-9ba8-14c094e16c20",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "  \"TABLE_NAME\"\n",
+    "FROM \"INFORMATION_SCHEMA\".\"TABLES\"\n",
+    "WHERE \"TABLE_SCHEMA\" = 'lookup'\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7518d6e2-4c24-4ced-bcbf-7982877d3b83",
+   "metadata": {},
+   "source": [
+    "Run the following cell to see how the [`LOOKUP`](https://druid.apache.org/docs/27.0.0/querying/sql-functions#lookup) SQL function can also be used as an alternative to  `JOIN`, in the case of this example, could have been more complicated."
```

Comment:
Not sure I follow this -- it seems like the JOIN query looks more complicated but I'm comparing these too simplistically. The results are also different; is that intentional?